aws-ts-netlify-cms-and-oauth/cms/infrastructure/index.ts

```diff
@@ -62,7 +71,7 @@ crawlDirectory(
                 key: relativeFilePath,
```

Comment:
Shouldn't this be `BucketObjectV2` as well?

Reply:
Not updating those, as there is a future for BucketObject maintenance. See comment in the other thread.